File: python/1101_sequencia_de_numeros_e_soma.py
resultados = []

while True:
    N, M = map(int, input().split())

    if N <= 0 or M <= 0:
        break

    if N > M:
        N, M = M, N

    seq = range(N, M+1)

    soma = sum(seq)

    resultados.append(f"{' '.join(map(str, seq))} Sum={soma}")

for resultado in resultados:
    print(f'{resultado}')

File: c/c99/1 - Iniciante/1101_sequencia_de_numeros_e_soma.c
#include <stdio.h>

int main() {
    int M, N;

    while (1) {
        scanf("%d %d", &M, &N);

        if (M <= 0 || N <= 0) {
            break;
        }

        int soma = 0;

        if (M > N) {
            int temp = M;
            M = N;
            N = temp;
        }

        for (int i = M; i <= N; i++) {
            printf("%d ", i);
            soma += i;
        }
        printf("Sum=%d\n", soma);
    }

    return 0;
}
